teamA/jeongDoYoung/app.js

```diff
@@ -0,0 +1,35 @@
+
+//built-in package
+const http = require("http");
+//3rd-party package
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const dotenv = require("dotenv").config();
```

Comment:
- 해당 부분은 아래와 같이 수정한 후 최상단에 올려주세요 !
```suggestion
require("dotenv").config();
```